testsuite/tests/ghci/scripts/T8959b.stderr

```diff
@@ -12,5 +12,5 @@ T8959b.hs:8:7: error:
 T8959b.hs:10:7: error:
     • Couldn't match expected type ‘(∀ a2. a2 → a2) → a1’
                   with actual type ‘()’
-    • In the expression: () ∷ (∀ a. a -> a) -> a
-      In an equation for ‘baz’: baz = () ∷ (∀ a. a -> a) -> a
+    • In the expression: () ∷ (∀ a. a → a) → a
+      In an equation for ‘baz’: baz = () ∷ (∀ a. a → a) → a
```

Comment:
Is this expected?

Reply:
Yes, I think this should be the expected behavior and this is a bug in master. The test uses -fprint-unicode-syntax.

Reply:
Submitted as https://phabricator.haskell.org/D5439.